src/QPConstr.cpp

```diff
@@ -77,6 +93,9 @@ JointLimitsConstr::JointLimitsConstr(const std::vector<rbd::MultiBody> & mbs,
 
   rbd::paramToVector(aDBound.lAlphaDBound, alphaDLower_);
   rbd::paramToVector(aDBound.uAlphaDBound, alphaDUpper_);
+
+  rbd::paramToVector(aDDBound.lAlphaDDBound, alphaDDLower_);
+  rbd::paramToVector(aDDBound.uAlphaDDBound, alphaDDUpper_);
```

Comment:
I think you can pre-multiply `alphaDDLower_` and `alphaDDUpper_` by `step_` here